test-packages/example.multipleeps/setup.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from setuptools import find_packages, setup
+
+
+setup(
+    name="example.multipleeps",
+    version="1.0a1",
+    description="An add-on for Plone",
+    long_description="long_description",
+    author="Thomas Schorr",
+    author_email="t_schorr@gmx.de",
+    license="GPL version 2",
+    packages=find_packages("src", exclude=["ez_setup"]),
+    namespace_packages=["example"],
+    package_dir={"": "src"},
+    include_package_data=True,
+    zip_safe=False,
+    install_requires=[
+        "setuptools",
+    ],
+    entry_points={
+        'z3c.autoinclude.plugin': [
+            'dummy = dummy',
+        ],
+        'plone.autoinclude.plugin': [
+            'target = plone',
+            'module = example.multipleeps',
+            'zcml = configure.zcml',
```

Comment:
The code does not do anything yet with the `zcml` option. But we might change that, we indeed had some ideas. Might be fine to save this until a 1.1 version.
But fine to have it in here already.